az-core/src/main/java/azkaban/Constants.java

```diff
@@ -496,5 +497,18 @@
         AZKABAN_CONTAINERIZED_PREFIX + "execution.processing.thread.pool.size";
     public static final String CONTAINERIZED_CREATION_RATE_LIMIT =
         AZKABAN_CONTAINERIZED_PREFIX + "creation.rate.limit";
+    public static final String AZKABAN_KUBERNETES_PREFIX="azkaban.kubernetes.";
+    public static final String KUBERNETES_SERVICE_REQUIRED = AZKABAN_KUBERNETES_PREFIX +
+        "service.required";
+    public static final String KUBERNETES_NAMESPACE = AZKABAN_KUBERNETES_PREFIX + "namespace";
+    public static final String KUBERNETES_KUBE_CONFIG_PATH = AZKABAN_KUBERNETES_PREFIX +
+        "kube.config.path";
+    public static final String KUBERNETES_POD_NAME_PREFIX = AZKABAN_KUBERNETES_PREFIX + "pod.name"
+        + ".prefix";
+    public static final String KUBERNETES_SERVICE_NAME_PREFIX = AZKABAN_KUBERNETES_PREFIX +
```

Comment:
pod and flow seem like prefixes for which more things properties can show up over time. I think it would be better to have "pod" and "flow" keywords get their own mnemonics.

Reply:
Thank you. Took care of it.